docs/UserGuide.md

```diff
@@ -352,6 +353,7 @@ Format: `rsvsort`
 * Upcoming reservations are always on top of expired reservations.
 * Upcoming reservations are sorted from earliest to latest.
 * Expired reservations are also sorted from earliest to latest.
+* If you add an upcoming reservation and it becomes expired while the application is open, `rsvsort` will update the reservation card to show the expiry flag.
```

Comment:
* Might be clearer to user if we add "calling the `rsvsort` command **again** will..."
* Might also want to split this sentence into two halves:
  1. If you add an upcoming reservation and it becomes expired while the application is open, the expiry tag will not be updated automatically.
  2. You will need to call the `rsvsort` command **again** to view the most correct and updated expiry flags.